Tests/SwiftDocCTests/Semantics/SymbolTests.swift

```diff
@@ -1111,7 +1112,120 @@ class SymbolTests: XCTestCase {
         let _ = DocumentationNode.contentFrom(documentedSymbol: symbol, documentationExtension: nil, engine: engine)
         XCTAssertEqual(engine.problems.count, 0)
     }
-    
+
+    func testAddingConstraintsToSymbol() throws {
+        let (withoutArticle, _) = try makeDocumentationNodeSymbol(
+            docComment: """
+                A cool API to call.
+
+                - Parameters:
+                  - name: A parameter
+                - Returns: Return value
+                """,
+            articleContent: nil
+        )
+
+        // This symbol starts with 3 generic constraints. See:
+
+        // jq < "Tests/SwiftDocCTests/Test Bundles/TestBundle.docc/mykit-iOS.symbols.json" '.symbols[] | select(.identifier.precise == "s:5MyKit0A5ClassC10myFunctionyyF") | .swiftExtension'
```

Comment:
I would maybe just remove the "jq" part of the comment. 

```suggestion
        // The original symbol has 3 Swift extension constraints:
```